src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -372,7 +367,7 @@ const submitEmailModeForm: ControllerHandler<
           )
           // MyInfo access token is single-use, so clear it
           return res
-            .clearCookie(MYINFO_COOKIE_NAME, MYINFO_COOKIE_OPTIONS)
+            .clearCookie(MYINFO_LOGIN_COOKIE_NAME, MYINFO_LOGIN_COOKIE_OPTIONS)
```

Comment:
comment: Feel like we clear many times, but i guess it doesn't hurt.

Reply:
hmm, maybe it seems that way because now there are two cookies, so we call `clearCookie` twice at the `/myinfo/login` endpoint. actually this is the only call to `clearCookie` at this endpoint, so I don't think this one is avoidable. for the other one, I've already reduced it to one `clearCookie` for each cookie type, which should be the minimum.